pytest-wdl-tests/tests/expr/types/coercion/identity_coercions/test_identity_coercions.wdl

```diff
@@ -0,0 +1,54 @@
+version 1.0
+
+workflow test_identity_coercions {
+    input {
+        String wf_in_file_name
+        String wf_in_file_content
+
+        String in_string
+        Int in_int
+        Float in_float
+        Boolean in_boolean
+        Array[Int] in_array
+        Map[Int, String] in_map
+    }
+
+    String string = in_string
+    Int int = in_int
+    Float float = in_float
+    Boolean boolean = in_boolean
+    Array[Int] array = in_array
+    Map[String, String] map = in_map
```

Comment:
Can you replace the `map` variable with, say, `xmap`, or something that isn't just `map`. There are languages where it is a reserved word.